ipatests/test_integration/test_random_serial_numbers.py

```diff
@@ -65,8 +69,47 @@ def install(cls, mh):
             raise pytest.skip("RNSv3 not supported")
         super(TestServerCALessToExternalCA_RSN, cls).install(mh)
 
+
+class TestStarTopo_RSN(IntegrationTest):
+    random_serial = True
+    topology = "star"
+
     @classmethod
-    def uninstall(cls, mh):
+    def install(cls, mh):
+        if not pki_supports_RSNv3(mh.master):
+            raise pytest.skip("RNSv3 not supported")
+        tasks.install_topo(
+            cls.topology, cls.master, cls.replicas, cls.clients
+        )
+
+
+class TestLineTopo_RSN(IntegrationTest):
+    random_serial = True
+    num_replicas = 2
+    topology = "line"
+
+    @classmethod
+    def install(cls, mh):
+        if not pki_supports_RSNv3(mh.master):
+            raise pytest.skip("RNSv3 not supported")
+        super(TestLineTopo_RSN, cls).install(mh)
+
+
+class TestRSNPKIConfig(IntegrationTest):
+    random_serial = True
+    num_replicas = 1
+
+    @classmethod
+    def install(cls, mh):
         if not pki_supports_RSNv3(mh.master):
             raise pytest.skip("RSNv3 not supported")
-        super(TestServerCALessToExternalCA_RSN, cls).uninstall(mh)
+        super(TestRSNPKIConfig, cls).install(mh)
+
+    def check_pki_config_params(self):
```

Comment:
The method needs to be named test_something otherwise it’s not considered as a test